cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -45,7 +45,10 @@ spec:
     # mirroring mode: pool level or per image
     # for more details see: https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring
     mode: image
-  # reports pool mirroring status if enabled
+    # snapshotSchedules list of snapshot schedules
+    snapshotSchedules:
```

Comment:
Do we really want to enable the snapshot schedule by default in our main example? Or should we comment this out?

Reply:
Comment is probably better.

Reply:
But because mirroring is not enable this won't do anything, but still I'll comment it.